Automattic/sensei
# Add toolbar pagination settings

Fixes #4389

![expensify](https://user-images.githubusercontent.com/53191348/142436244-b6efe467-95d2-4ffd-8214-20e865eb9e2a.gif)

### Changes proposed in this Pull Request

* Adds the settings in the quiz toolbar.
* Adds some extra validation like:
    * Do not allow the pagination value to be equal to the question count.
    * Disable the buttons when there are 0 or 1 questions to the quiz.
    * Hide the multi-page inputs when 'Single Page' is selected.

### Testing instructions
* Check the pagination controls in the quiz toolbar.
* Load a lesson with no quiz or with a quiz with the single questions and observe that the controls are disabled.
* Update the pagination value of the quiz and observe that it is stored correctly.
* Try adding/removing questions in the quiz and observe that the controls are updated appropriately.
